scripts/packages/README

```diff
@@ -443,6 +443,12 @@ recv_addr
     turns on incremental state transfer. IST will use SSL if SSL is configured
     as described above. No default.
 
+recv_bind
```

Comment:
I retract my comment previous here. Of course recv_bind is the proper name. Sorry.